chapters/fundamentals.txt

```diff
@@ -885,6 +885,18 @@ knowledge of the application, without allowing return of values not defined
 by any extension.
 ====
 
+[NOTE]
+.Note
+====
+Application developers are encouraged to be careful when using `switch`
+statements with Vulkan API enums.
+This is because new extensions can add new values to existing enums.
+The use of a `default:` statement, within a `switch`, may avoid future
+compilation issues.
+Particularly for enums like e.g. elink:VkDriverId that may change at any
```

Comment:
Any of them can change any time an extension is added, so this sentence feels redundant.

Reply:
Right, I mean this one can change without enabling an extension.